packages/instrumentation/src/ActiveTracingHelper.ts

```diff
@@ -100,6 +109,11 @@ export class ActiveTracingHelper implements TracingHelper {
     const tracer = trace.getTracer('prisma')
     const context = options.context ?? this.getActiveContext()
     const name = `prisma:client:${options.name}`
+
+    if (this.ignoreLayersTypes.includes(name)) {
```

Comment:
More of a conceptual question rather than request to change: should ignoring the span also ignore all of it's children? If so, i think we'll need to suppress tracing for the callback.
See, for example, [how http instrumentation does it](https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/opentelemetry-instrumentation-http/src/http.ts#L494).

Problem here is that it won't work for engine spans (see #17953), but that's something for us to solve, not for this PR.

Reply:
That is a great question. One scenario is related to the need to trace the query information without the data related to connection, serialisation and co. 
If we follow the same approach of http instrumentation, we would need to update the prisma layers to allow a more flexible configuration.

Personally, the most useful span related to an ORM is wrapping the full ORM logic and presenting query data (keeping PII obfuscation in mind) considering that tools like AWS X-Ray will only display traces with up to 200 segments